ui/apps/platform/src/Containers/MainPage/Navigation/HorizontalSubnav.tsx

```diff
@@ -106,11 +150,13 @@ function getSubnavGroupForCurrentPath(
         Object.values(subnavDescriptionGroups).find((subnavDescriptionGroup) => {
             return subnavDescriptionGroup.some((group) => {
                 if (group.type === 'link') {
-                    return matchPath(pathname, group.path);
+                    return matchBasePath({ pathname, descriptionPath: group.path });
```

Comment:
What are the benefits of creating an object parameter instead of following the same convention as "matchPath" in this situation?

Reply:
I went with the named parameters via object properties due to both values being of type `string` and both having similar semantics. In the `matchPath` example here, we are passing two strings `pathname` and `group.path`. Without more context, it is a little difficult to tell how these two paths differ and would be easy to accidentally change the ordering when calling the function.

With the named object properties pattern, we do not have to worry about `string` positioning and we have a little more flexibility to name the params closer to what they actually are.